### DISCUSSION TITLE
Finding a good layout - Psychiatric Room

### DISCUSSION THREAD

**Discussion by ageekhere** [CorsixTH/CorsixTH#2423] [General Diagnostics]
I am exploring good design layouts for each room, in this post I am focusing on the Psychiatric Room.

The Goal
Keeping the room as small as possible while keeping the doctor as happy as possible(avoid being unhappy and wanting a pay rise) while optimizing turnaround time for handyman plant watering and patient walk times.

**Version 1.00**
![Screenshot 2023-09-30 130316](https://github.com/CorsixTH/CorsixTH/assets/13173133/d36c7cee-3bc4-4e52-bdda-7c412465d11e)

**Items**
1x Screen
1x Couch
1x Skeleton
1x Comfortable Chair
1x Bookcase
1x Plant
1x Bin
1x Fire extinguisher
1x Radiator

**Summary**
This room is complex as there are lots of items to fit in the room. First the couch needs to be as near to the door as possible while also being near the Screen. In design 1.00 because of the Couch and Screen placement the Plant cannot be next to the door. Fire extinguisher is in center of the room. The Chair, Bookcase and Skeleton are right next to each other for fast access for the doctor.

What is unclear 
What does the Skeleton do in the Psychiatric?
Can the patient still access the bin?
Is another Radiator needed?

Lets start the decision, can this design be improved and if so how?

**Version 1.01**
added @PaveMentman Improvements to have plant by door.
![Screenshot 2023-10-02 083335](https://github.com/CorsixTH/CorsixTH/assets/13173133/bf195cde-f106-48c6-b037-ccae8bef72d0)

**Comment by PaveMentman** [comment]
---
Psychiatric Room is yet another one worth having two of near the helipad for the emergencies.

And once again we the Screen to take into account.


I've observed the Skeleton is only used during diagnosis-visit so most likely it helps with the diagnosis-progression similar to how it is used in the Training Rooms.
How many points this item improves is something I do not know.


Once again, radiator and fire-extinguisher I "ignore",
as I had already explained here:
https://github.com/CorsixTH/CorsixTH/discussions/2430#discussioncomment-7157735

![2023-10-01-17-09-57-651-CorsixTH-CorsixTH](https://github.com/CorsixTH/CorsixTH/assets/6709463/1eb3ed44-91fe-4a32-be5a-a374b38d0daa)
![2023-10-01-17-10-00-184-CorsixTH-CorsixTH](https://github.com/CorsixTH/CorsixTH/assets/6709463/f37300b1-b26a-4dbd-b546-1099aa5f316c)


---

**Reply by ageekhere** [reply]
I like how in this design the plant is next to the door while still having the same walk paths for doctor and patient. I think i may have found a better location for the bin. look at version 1.01.
